Remove Windows WebKitLegacy code

brentfulgham opened the pull request:
#### 252ee7ef89ec4685d4c20b12b5f6dba0795988db
<pre>
Remove Windows WebKitLegacy code
<a href="https://bugs.webkit.org/show_bug.cgi?id=252336">https://bugs.webkit.org/show_bug.cgi?id=252336</a>
&lt;rdar://problem/105510314&gt;

Reviewed by Fujii Hironori.

WebKitLegacy code is no longer used on any Windows port. We can now delete
the relevant code.

This patch moves some resource files from WebKitLegacy to WinCairo&apos;s platform folder,
since they are needed for some UI features.

* Source/WebCore/platform/win/WebCore.resources/WebKit.rc: Renamed from Source/WebKitLegacy/win/WebKit.resources/WebKit.rc.
* Source/WebCore/platform/win/WebCore.resources/deleteButton.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/deleteButton.png.
* Source/WebCore/platform/win/WebCore.resources/deleteButtonPressed.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/deleteButtonPressed.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoAudioVolumeHigh.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoAudioVolumeHigh.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoAudioVolumeLow.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoAudioVolumeLow.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoExitFullscreen.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoExitFullscreen.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoPause.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoPause.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoPlay.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoPlay.png.
* Source/WebCore/platform/win/WebCore.resources/missingImage.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/missingImage.png.
* Source/WebCore/platform/win/WebCore.resources/nullplugin.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/nullplugin.png.
* Source/WebCore/platform/win/WebCore.resources/panEastCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panEastCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panIcon.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panIcon.png.
* Source/WebCore/platform/win/WebCore.resources/panNorthCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panNorthCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panNorthEastCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panNorthEastCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panNorthWestCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panNorthWestCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panSouthCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panSouthCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panSouthEastCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panSouthEastCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panSouthWestCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panSouthWestCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panWestCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panWestCursor.png.
* Source/WebCore/platform/win/WebCore.resources/searchCancel.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/searchCancel.png.
* Source/WebCore/platform/win/WebCore.resources/searchCancelPressed.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/searchCancelPressed.png.
* Source/WebCore/platform/win/WebCore.resources/searchMagnifier.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/searchMagnifier.png.
* Source/WebCore/platform/win/WebCore.resources/searchMagnifierResults.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/searchMagnifierResults.png.
* Source/WebCore/platform/win/WebCore.resources/textAreaResizeCorner.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/textAreaResizeCorner.png.
* Source/WebCore/platform/win/WebCore.resources/verticalTextCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/verticalTextCursor.png.
* Source/WebCore/platform/win/WebCore.resources/zoomInCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/zoomInCursor.png.
* Source/WebCore/platform/win/WebCore.resources/zoomOutCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/zoomOutCursor.png.
* Source/WebCore/platform/win/resource.h: Renamed from Source/WebKitLegacy/win/resource.h.
* Source/WebKitLegacy/CMakeLists.txt:
* Source/WebKitLegacy/PlatformWin.cmake: Removed.
* Source/WebKitLegacy/WebKitLegacy.vcxproj/WebKitLegacy.proj: Removed.
* Source/WebKitLegacy/cf/WebCoreSupport/WebInspectorClientCF.cpp:
* Source/WebKitLegacy/win/AccessibleBase.cpp: Removed.
* Source/WebKitLegacy/win/AccessibleBase.h: Removed.
* Source/WebKitLegacy/win/AccessibleDocument.cpp: Removed.
* Source/WebKitLegacy/win/AccessibleDocument.h: Removed.
* Source/WebKitLegacy/win/AccessibleImage.cpp: Removed.
* Source/WebKitLegacy/win/AccessibleImage.h: Removed.
* Source/WebKitLegacy/win/AccessibleTextImpl.cpp: Removed.
* Source/WebKitLegacy/win/AccessibleTextImpl.h: Removed.
* Source/WebKitLegacy/win/BackForwardList.cpp: Removed.
* Source/WebKitLegacy/win/BackForwardList.h: Removed.
* Source/WebKitLegacy/win/CFDictionaryPropertyBag.cpp: Removed.
* Source/WebKitLegacy/win/CFDictionaryPropertyBag.h: Removed.
* Source/WebKitLegacy/win/COMEnumVariant.h: Removed.
* Source/WebKitLegacy/win/COMPropertyBag.h: Removed.
* Source/WebKitLegacy/win/COMVariantSetter.h: Removed.
* Source/WebKitLegacy/win/CodeAnalysisConfig.h: Removed.
* Source/WebKitLegacy/win/DOMCSSClasses.cpp: Removed.
* Source/WebKitLegacy/win/DOMCSSClasses.h: Removed.
* Source/WebKitLegacy/win/DOMCoreClasses.cpp: Removed.
* Source/WebKitLegacy/win/DOMCoreClasses.h: Removed.
* Source/WebKitLegacy/win/DOMEventsClasses.cpp: Removed.
* Source/WebKitLegacy/win/DOMEventsClasses.h: Removed.
* Source/WebKitLegacy/win/DOMHTMLClasses.cpp: Removed.
* Source/WebKitLegacy/win/DOMHTMLClasses.h: Removed.
* Source/WebKitLegacy/win/DefaultDownloadDelegate.cpp: Removed.
* Source/WebKitLegacy/win/DefaultDownloadDelegate.h: Removed.
* Source/WebKitLegacy/win/DefaultPolicyDelegate.cpp: Removed.
* Source/WebKitLegacy/win/DefaultPolicyDelegate.h: Removed.
* Source/WebKitLegacy/win/ForEachCoClass.cpp: Removed.
* Source/WebKitLegacy/win/ForEachCoClass.h: Removed.
* Source/WebKitLegacy/win/FullscreenVideoController.cpp: Removed.
* Source/WebKitLegacy/win/FullscreenVideoController.h: Removed.
* Source/WebKitLegacy/win/Interfaces/AccessibilityDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/Accessible2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/Accessible2_2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleApplication.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleEditableText.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleRelation.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleStates.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleText.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleText2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/IA2CommonTypes.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/AccessibleComparable.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMCSS.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMCore.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMEvents.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMExtensions.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMHTML.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMRange.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMWindow.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IGEN_DOMObject.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebApplicationCache.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebArchive.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebBackForwardList.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebBackForwardListPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebCache.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebCoreStatistics.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDataSource.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDatabaseManager.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDesktopNotificationsDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDocument.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDownload.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebEditingDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebEmbeddedView.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebError.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebErrorPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFormDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrame.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrameLoadDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrameLoadDelegatePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrameLoadDelegatePrivate2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFramePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrameView.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebGeolocationPolicyListener.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebGeolocationPosition.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebGeolocationProvider.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHTMLRepresentation.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHTTPURLResponse.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistory.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistoryDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistoryItem.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistoryItemPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistoryPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebInspector.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebInspectorPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebJavaScriptCollector.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebKitStatistics.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebMutableURLRequest.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebMutableURLRequestPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebNavigationData.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebNotification.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebNotificationCenter.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebNotificationObserver.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebPolicyDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebPolicyDelegatePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebPreferences.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebPreferencesPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebResource.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebResourceLoadDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebResourceLoadDelegatePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebResourceLoadDelegatePrivate2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebScriptObject.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebScriptWorld.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebSecurityOrigin.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebSerializedJSValue.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebSerializedJSValuePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebTextRenderer.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUIDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUIDelegate2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUIDelegatePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebURLAuthenticationChallenge.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebURLRequest.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebURLResponse.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebURLResponsePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUndoManager.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUndoTarget.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUserContentURLPattern.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebView.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebViewPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebWorkersPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/JavaScriptCoreAPITypes.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/WebKit.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/WebKitMessageLoop.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/WebScrollbarTypes.idl: Removed.
* Source/WebKitLegacy/win/MarshallingHelpers.cpp: Removed.
* Source/WebKitLegacy/win/MarshallingHelpers.h: Removed.
* Source/WebKitLegacy/win/MemoryStream.cpp: Removed.
* Source/WebKitLegacy/win/MemoryStream.h: Removed.
* Source/WebKitLegacy/win/ProgIDMacros.h: Removed.
* Source/WebKitLegacy/win/Scripts/PreferencesTemplates/WebPreferencesDefinitions.h.erb: Removed.
* Source/WebKitLegacy/win/Scripts/PreferencesTemplates/WebViewPreferencesChangedGenerated.cpp.erb: Removed.
* Source/WebKitLegacy/win/WebActionPropertyBag.cpp: Removed.
* Source/WebKitLegacy/win/WebActionPropertyBag.h: Removed.
* Source/WebKitLegacy/win/WebApplicationCache.cpp: Removed.
* Source/WebKitLegacy/win/WebApplicationCache.h: Removed.
* Source/WebKitLegacy/win/WebArchive.cpp: Removed.
* Source/WebKitLegacy/win/WebArchive.h: Removed.
* Source/WebKitLegacy/win/WebBackForwardList.cpp: Removed.
* Source/WebKitLegacy/win/WebBackForwardList.h: Removed.
* Source/WebKitLegacy/win/WebCache.cpp: Removed.
* Source/WebKitLegacy/win/WebCache.h: Removed.
* Source/WebKitLegacy/win/WebCachedFramePlatformData.h: Removed.
* Source/WebKitLegacy/win/WebCoreStatistics.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreStatistics.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/AcceleratedCompositingContext.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/AcceleratedCompositingContext.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebChromeClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebChromeClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebContextMenuClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebContextMenuClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebDesktopNotificationsDelegate.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebDesktopNotificationsDelegate.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebDragClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebDragClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebEditorClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebEditorClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebFrameLoaderClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebFrameLoaderClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebFrameNetworkingContext.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebFrameNetworkingContext.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebGeolocationClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebGeolocationClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebInspectorClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebInspectorClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebInspectorDelegate.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebInspectorDelegate.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebPlatformStrategies.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebPlatformStrategies.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebPluginInfoProvider.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebPluginInfoProvider.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebProgressTrackerClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebProgressTrackerClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebVisitedLinkStore.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebVisitedLinkStore.h: Removed.
* Source/WebKitLegacy/win/WebDataSource.cpp: Removed.
* Source/WebKitLegacy/win/WebDataSource.h: Removed.
* Source/WebKitLegacy/win/WebDatabaseManager.cpp: Removed.
* Source/WebKitLegacy/win/WebDatabaseManager.h: Removed.
* Source/WebKitLegacy/win/WebDocumentLoader.cpp: Removed.
* Source/WebKitLegacy/win/WebDocumentLoader.h: Removed.
* Source/WebKitLegacy/win/WebDownload.cpp: Removed.
* Source/WebKitLegacy/win/WebDownload.h: Removed.
* Source/WebKitLegacy/win/WebDownloadCFNet.cpp: Removed.
* Source/WebKitLegacy/win/WebDownloadCurl.cpp: Removed.
* Source/WebKitLegacy/win/WebDropSource.cpp: Removed.
* Source/WebKitLegacy/win/WebDropSource.h: Removed.
* Source/WebKitLegacy/win/WebElementPropertyBag.cpp: Removed.
* Source/WebKitLegacy/win/WebElementPropertyBag.h: Removed.
* Source/WebKitLegacy/win/WebError.cpp: Removed.
* Source/WebKitLegacy/win/WebError.h: Removed.
* Source/WebKitLegacy/win/WebFrame.cpp: Removed.
* Source/WebKitLegacy/win/WebFrame.h: Removed.
* Source/WebKitLegacy/win/WebFramePolicyListener.cpp: Removed.
* Source/WebKitLegacy/win/WebFramePolicyListener.h: Removed.
* Source/WebKitLegacy/win/WebGeolocationPolicyListener.cpp: Removed.
* Source/WebKitLegacy/win/WebGeolocationPolicyListener.h: Removed.
* Source/WebKitLegacy/win/WebGeolocationPosition.cpp: Removed.
* Source/WebKitLegacy/win/WebGeolocationPosition.h: Removed.
* Source/WebKitLegacy/win/WebHTMLRepresentation.cpp: Removed.
* Source/WebKitLegacy/win/WebHTMLRepresentation.h: Removed.
* Source/WebKitLegacy/win/WebHistory.cpp: Removed.
* Source/WebKitLegacy/win/WebHistory.h: Removed.
* Source/WebKitLegacy/win/WebHistoryItem.cpp: Removed.
* Source/WebKitLegacy/win/WebHistoryItem.h: Removed.
* Source/WebKitLegacy/win/WebInspector.cpp: Removed.
* Source/WebKitLegacy/win/WebInspector.h: Removed.
* Source/WebKitLegacy/win/WebJavaScriptCollector.cpp: Removed.
* Source/WebKitLegacy/win/WebJavaScriptCollector.h: Removed.
* Source/WebKitLegacy/win/WebKit.resources/Info.plist: Removed.
* Source/WebKitLegacy/win/WebKitCOMAPI.cpp: Removed.
* Source/WebKitLegacy/win/WebKitCOMAPI.h: Removed.
* Source/WebKitLegacy/win/WebKitClassFactory.cpp: Removed.
* Source/WebKitLegacy/win/WebKitClassFactory.h: Removed.
* Source/WebKitLegacy/win/WebKitDLL.cpp: Removed.
* Source/WebKitLegacy/win/WebKitDLL.h: Removed.
* Source/WebKitLegacy/win/WebKitGraphics.cpp: Removed.
* Source/WebKitLegacy/win/WebKitGraphics.h: Removed.
* Source/WebKitLegacy/win/WebKitLogging.cpp: Removed.
* Source/WebKitLegacy/win/WebKitLogging.h: Removed.
* Source/WebKitLegacy/win/WebKitMessageLoop.cpp: Removed.
* Source/WebKitLegacy/win/WebKitMessageLoop.h: Removed.
* Source/WebKitLegacy/win/WebKitPrefix.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFImage.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFImage.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFImageInternal.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFView.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFView.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFViewPrivate.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WebKitQuartzCoreAdditions.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WebKitQuartzCoreAdditionsBase.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CAD3DRenderer.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CAD3DRenderer.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CAView.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CAView.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CVDisplayLink.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CVDisplayLink.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CVDisplayLinkClient.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/Image.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/Image.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/ImageConversion.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/ImageConversion.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/InvertColorPS.hlsl: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/WebKitQuartzCoreAdditionsInternal.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/WebKitQuartzCoreAdditionsPrefix.h: Removed.
* Source/WebKitLegacy/win/WebKitStatistics.cpp: Removed.
* Source/WebKitLegacy/win/WebKitStatistics.h: Removed.
* Source/WebKitLegacy/win/WebKitStatisticsPrivate.h: Removed.
* Source/WebKitLegacy/win/WebKitSystemBits.cpp: Removed.
* Source/WebKitLegacy/win/WebKitSystemBits.h: Removed.
* Source/WebKitLegacy/win/WebLocalizableStrings.cpp: Removed.
* Source/WebKitLegacy/win/WebLocalizableStrings.h: Removed.
* Source/WebKitLegacy/win/WebMutableURLRequest.cpp: Removed.
* Source/WebKitLegacy/win/WebMutableURLRequest.h: Removed.
* Source/WebKitLegacy/win/WebNavigationData.cpp: Removed.
* Source/WebKitLegacy/win/WebNavigationData.h: Removed.
* Source/WebKitLegacy/win/WebNodeHighlight.cpp: Removed.
* Source/WebKitLegacy/win/WebNodeHighlight.h: Removed.
* Source/WebKitLegacy/win/WebNotification.cpp: Removed.
* Source/WebKitLegacy/win/WebNotification.h: Removed.
* Source/WebKitLegacy/win/WebNotificationCenter.cpp: Removed.
* Source/WebKitLegacy/win/WebNotificationCenter.h: Removed.
* Source/WebKitLegacy/win/WebPreferenceKeysPrivate.h: Removed.
* Source/WebKitLegacy/win/WebPreferences.cpp: Removed.
* Source/WebKitLegacy/win/WebPreferences.h: Removed.
* Source/WebKitLegacy/win/WebResource.cpp: Removed.
* Source/WebKitLegacy/win/WebResource.h: Removed.
* Source/WebKitLegacy/win/WebScriptObject.cpp: Removed.
* Source/WebKitLegacy/win/WebScriptObject.h: Removed.
* Source/WebKitLegacy/win/WebScriptWorld.cpp: Removed.
* Source/WebKitLegacy/win/WebScriptWorld.h: Removed.
* Source/WebKitLegacy/win/WebSecurityOrigin.cpp: Removed.
* Source/WebKitLegacy/win/WebSecurityOrigin.h: Removed.
* Source/WebKitLegacy/win/WebSerializedJSValue.cpp: Removed.
* Source/WebKitLegacy/win/WebSerializedJSValue.h: Removed.
* Source/WebKitLegacy/win/WebTextRenderer.cpp: Removed.
* Source/WebKitLegacy/win/WebTextRenderer.h: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallenge.cpp: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallenge.h: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallengeSender.cpp: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallengeSender.h: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallengeSenderCFNet.cpp: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallengeSenderCurl.cpp: Removed.
* Source/WebKitLegacy/win/WebURLCredential.cpp: Removed.
* Source/WebKitLegacy/win/WebURLCredential.h: Removed.
* Source/WebKitLegacy/win/WebURLProtectionSpace.cpp: Removed.
* Source/WebKitLegacy/win/WebURLProtectionSpace.h: Removed.
* Source/WebKitLegacy/win/WebURLResponse.cpp: Removed.
* Source/WebKitLegacy/win/WebURLResponse.h: Removed.
* Source/WebKitLegacy/win/WebUserContentURLPattern.cpp: Removed.
* Source/WebKitLegacy/win/WebUserContentURLPattern.h: Removed.
* Source/WebKitLegacy/win/WebView.cpp: Removed.
* Source/WebKitLegacy/win/WebView.h: Removed.
* Source/WebKitLegacy/win/WebWorkersPrivate.cpp: Removed.
* Source/WebKitLegacy/win/WebWorkersPrivate.h: Removed.
* Source/WebKitLegacy/win/storage/WebDatabaseProvider.cpp: Removed.

Canonical link: <a href="https://commits.webkit.org/260397@main">https://commits.webkit.org/260397@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8e94d08d326cad8f3b0512541fc4038440219521

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/108174 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/17256 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/41041 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/117298 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/116615 "Built successfully") 
| | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/76/builds/18649 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/8542 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/100381 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/113941 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/14027 "Build was cancelled. Recent messages:Encountered some issues during cleanup") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/97254 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/41966 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/95938 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/28894 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/83637 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/1/builds/97393 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/10122 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/30241 "Passed tests") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/38/builds/96751 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/83/builds/8224 "Built successfully and passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/10845 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/7143 "Passed tests") | [  ~~🧪 jsc-armv7-tests~~](https://ews-build.webkit.org/#/builders/46/builds/30925 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/16270 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/49833 "Passed tests") | [✅ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/37/builds/105778 "Built successfully") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7194 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/12427 "Built successfully") | | [✅ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/45/builds/26186 "Passed tests") | 
| | | | | 
<!--EWS-Status-Bubble-End-->

fujii reviewed (changes requested):
Please just remove the icons. Some of them are obsolete. I will check and readd later.

brentfulgham committed:
Remove Windows WebKitLegacy code
https://bugs.webkit.org/show_bug.cgi?id=252336
<rdar://problem/105510314>

Reviewed by Fujii Hironori.

WebKitLegacy code is no longer used on any Windows port. We can now delete
the relevant code.

This patch moves some resource files from WebKitLegacy to WinCairo's platform folder,
since they are needed for some UI features.

* Source/WebCore/platform/win/WebCore.resources/WebKit.rc: Renamed from Source/WebKitLegacy/win/WebKit.resources/WebKit.rc.
* Source/WebCore/platform/win/WebCore.resources/deleteButton.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/deleteButton.png.
* Source/WebCore/platform/win/WebCore.resources/deleteButtonPressed.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/deleteButtonPressed.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoAudioVolumeHigh.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoAudioVolumeHigh.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoAudioVolumeLow.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoAudioVolumeLow.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoExitFullscreen.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoExitFullscreen.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoPause.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoPause.png.
* Source/WebCore/platform/win/WebCore.resources/fsVideoPlay.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/fsVideoPlay.png.
* Source/WebCore/platform/win/WebCore.resources/missingImage.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/missingImage.png.
* Source/WebCore/platform/win/WebCore.resources/nullplugin.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/nullplugin.png.
* Source/WebCore/platform/win/WebCore.resources/panEastCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panEastCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panIcon.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panIcon.png.
* Source/WebCore/platform/win/WebCore.resources/panNorthCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panNorthCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panNorthEastCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panNorthEastCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panNorthWestCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panNorthWestCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panSouthCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panSouthCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panSouthEastCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panSouthEastCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panSouthWestCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panSouthWestCursor.png.
* Source/WebCore/platform/win/WebCore.resources/panWestCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/panWestCursor.png.
* Source/WebCore/platform/win/WebCore.resources/searchCancel.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/searchCancel.png.
* Source/WebCore/platform/win/WebCore.resources/searchCancelPressed.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/searchCancelPressed.png.
* Source/WebCore/platform/win/WebCore.resources/searchMagnifier.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/searchMagnifier.png.
* Source/WebCore/platform/win/WebCore.resources/searchMagnifierResults.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/searchMagnifierResults.png.
* Source/WebCore/platform/win/WebCore.resources/textAreaResizeCorner.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/textAreaResizeCorner.png.
* Source/WebCore/platform/win/WebCore.resources/verticalTextCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/verticalTextCursor.png.
* Source/WebCore/platform/win/WebCore.resources/zoomInCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/zoomInCursor.png.
* Source/WebCore/platform/win/WebCore.resources/zoomOutCursor.png: Renamed from Source/WebKitLegacy/win/WebKit.resources/zoomOutCursor.png.
* Source/WebCore/platform/win/resource.h: Renamed from Source/WebKitLegacy/win/resource.h.
* Source/WebKitLegacy/CMakeLists.txt:
* Source/WebKitLegacy/PlatformWin.cmake: Removed.
* Source/WebKitLegacy/WebKitLegacy.vcxproj/WebKitLegacy.proj: Removed.
* Source/WebKitLegacy/cf/WebCoreSupport/WebInspectorClientCF.cpp:
* Source/WebKitLegacy/win/AccessibleBase.cpp: Removed.
* Source/WebKitLegacy/win/AccessibleBase.h: Removed.
* Source/WebKitLegacy/win/AccessibleDocument.cpp: Removed.
* Source/WebKitLegacy/win/AccessibleDocument.h: Removed.
* Source/WebKitLegacy/win/AccessibleImage.cpp: Removed.
* Source/WebKitLegacy/win/AccessibleImage.h: Removed.
* Source/WebKitLegacy/win/AccessibleTextImpl.cpp: Removed.
* Source/WebKitLegacy/win/AccessibleTextImpl.h: Removed.
* Source/WebKitLegacy/win/BackForwardList.cpp: Removed.
* Source/WebKitLegacy/win/BackForwardList.h: Removed.
* Source/WebKitLegacy/win/CFDictionaryPropertyBag.cpp: Removed.
* Source/WebKitLegacy/win/CFDictionaryPropertyBag.h: Removed.
* Source/WebKitLegacy/win/COMEnumVariant.h: Removed.
* Source/WebKitLegacy/win/COMPropertyBag.h: Removed.
* Source/WebKitLegacy/win/COMVariantSetter.h: Removed.
* Source/WebKitLegacy/win/CodeAnalysisConfig.h: Removed.
* Source/WebKitLegacy/win/DOMCSSClasses.cpp: Removed.
* Source/WebKitLegacy/win/DOMCSSClasses.h: Removed.
* Source/WebKitLegacy/win/DOMCoreClasses.cpp: Removed.
* Source/WebKitLegacy/win/DOMCoreClasses.h: Removed.
* Source/WebKitLegacy/win/DOMEventsClasses.cpp: Removed.
* Source/WebKitLegacy/win/DOMEventsClasses.h: Removed.
* Source/WebKitLegacy/win/DOMHTMLClasses.cpp: Removed.
* Source/WebKitLegacy/win/DOMHTMLClasses.h: Removed.
* Source/WebKitLegacy/win/DefaultDownloadDelegate.cpp: Removed.
* Source/WebKitLegacy/win/DefaultDownloadDelegate.h: Removed.
* Source/WebKitLegacy/win/DefaultPolicyDelegate.cpp: Removed.
* Source/WebKitLegacy/win/DefaultPolicyDelegate.h: Removed.
* Source/WebKitLegacy/win/ForEachCoClass.cpp: Removed.
* Source/WebKitLegacy/win/ForEachCoClass.h: Removed.
* Source/WebKitLegacy/win/FullscreenVideoController.cpp: Removed.
* Source/WebKitLegacy/win/FullscreenVideoController.h: Removed.
* Source/WebKitLegacy/win/Interfaces/AccessibilityDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/Accessible2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/Accessible2_2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleApplication.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleEditableText.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleRelation.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleStates.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleText.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/AccessibleText2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/Accessible2/IA2CommonTypes.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/AccessibleComparable.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMCSS.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMCore.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMEvents.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMExtensions.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMHTML.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMRange.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/DOMWindow.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IGEN_DOMObject.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebApplicationCache.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebArchive.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebBackForwardList.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebBackForwardListPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebCache.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebCoreStatistics.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDataSource.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDatabaseManager.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDesktopNotificationsDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDocument.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebDownload.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebEditingDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebEmbeddedView.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebError.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebErrorPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFormDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrame.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrameLoadDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrameLoadDelegatePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrameLoadDelegatePrivate2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFramePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebFrameView.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebGeolocationPolicyListener.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebGeolocationPosition.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebGeolocationProvider.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHTMLRepresentation.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHTTPURLResponse.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistory.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistoryDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistoryItem.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistoryItemPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebHistoryPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebInspector.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebInspectorPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebJavaScriptCollector.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebKitStatistics.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebMutableURLRequest.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebMutableURLRequestPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebNavigationData.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebNotification.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebNotificationCenter.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebNotificationObserver.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebPolicyDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebPolicyDelegatePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebPreferences.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebPreferencesPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebResource.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebResourceLoadDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebResourceLoadDelegatePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebResourceLoadDelegatePrivate2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebScriptObject.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebScriptWorld.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebSecurityOrigin.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebSerializedJSValue.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebSerializedJSValuePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebTextRenderer.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUIDelegate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUIDelegate2.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUIDelegatePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebURLAuthenticationChallenge.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebURLRequest.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebURLResponse.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebURLResponsePrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUndoManager.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUndoTarget.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebUserContentURLPattern.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebView.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebViewPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/IWebWorkersPrivate.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/JavaScriptCoreAPITypes.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/WebKit.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/WebKitMessageLoop.idl: Removed.
* Source/WebKitLegacy/win/Interfaces/WebScrollbarTypes.idl: Removed.
* Source/WebKitLegacy/win/MarshallingHelpers.cpp: Removed.
* Source/WebKitLegacy/win/MarshallingHelpers.h: Removed.
* Source/WebKitLegacy/win/MemoryStream.cpp: Removed.
* Source/WebKitLegacy/win/MemoryStream.h: Removed.
* Source/WebKitLegacy/win/ProgIDMacros.h: Removed.
* Source/WebKitLegacy/win/Scripts/PreferencesTemplates/WebPreferencesDefinitions.h.erb: Removed.
* Source/WebKitLegacy/win/Scripts/PreferencesTemplates/WebViewPreferencesChangedGenerated.cpp.erb: Removed.
* Source/WebKitLegacy/win/WebActionPropertyBag.cpp: Removed.
* Source/WebKitLegacy/win/WebActionPropertyBag.h: Removed.
* Source/WebKitLegacy/win/WebApplicationCache.cpp: Removed.
* Source/WebKitLegacy/win/WebApplicationCache.h: Removed.
* Source/WebKitLegacy/win/WebArchive.cpp: Removed.
* Source/WebKitLegacy/win/WebArchive.h: Removed.
* Source/WebKitLegacy/win/WebBackForwardList.cpp: Removed.
* Source/WebKitLegacy/win/WebBackForwardList.h: Removed.
* Source/WebKitLegacy/win/WebCache.cpp: Removed.
* Source/WebKitLegacy/win/WebCache.h: Removed.
* Source/WebKitLegacy/win/WebCachedFramePlatformData.h: Removed.
* Source/WebKitLegacy/win/WebCoreStatistics.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreStatistics.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/AcceleratedCompositingContext.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/AcceleratedCompositingContext.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebChromeClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebChromeClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebContextMenuClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebContextMenuClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebDesktopNotificationsDelegate.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebDesktopNotificationsDelegate.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebDragClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebDragClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebEditorClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebEditorClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebFrameLoaderClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebFrameLoaderClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebFrameNetworkingContext.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebFrameNetworkingContext.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebGeolocationClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebGeolocationClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebInspectorClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebInspectorClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebInspectorDelegate.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebInspectorDelegate.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebPlatformStrategies.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebPlatformStrategies.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebPluginInfoProvider.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebPluginInfoProvider.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebProgressTrackerClient.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebProgressTrackerClient.h: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebVisitedLinkStore.cpp: Removed.
* Source/WebKitLegacy/win/WebCoreSupport/WebVisitedLinkStore.h: Removed.
* Source/WebKitLegacy/win/WebDataSource.cpp: Removed.
* Source/WebKitLegacy/win/WebDataSource.h: Removed.
* Source/WebKitLegacy/win/WebDatabaseManager.cpp: Removed.
* Source/WebKitLegacy/win/WebDatabaseManager.h: Removed.
* Source/WebKitLegacy/win/WebDocumentLoader.cpp: Removed.
* Source/WebKitLegacy/win/WebDocumentLoader.h: Removed.
* Source/WebKitLegacy/win/WebDownload.cpp: Removed.
* Source/WebKitLegacy/win/WebDownload.h: Removed.
* Source/WebKitLegacy/win/WebDownloadCFNet.cpp: Removed.
* Source/WebKitLegacy/win/WebDownloadCurl.cpp: Removed.
* Source/WebKitLegacy/win/WebDropSource.cpp: Removed.
* Source/WebKitLegacy/win/WebDropSource.h: Removed.
* Source/WebKitLegacy/win/WebElementPropertyBag.cpp: Removed.
* Source/WebKitLegacy/win/WebElementPropertyBag.h: Removed.
* Source/WebKitLegacy/win/WebError.cpp: Removed.
* Source/WebKitLegacy/win/WebError.h: Removed.
* Source/WebKitLegacy/win/WebFrame.cpp: Removed.
* Source/WebKitLegacy/win/WebFrame.h: Removed.
* Source/WebKitLegacy/win/WebFramePolicyListener.cpp: Removed.
* Source/WebKitLegacy/win/WebFramePolicyListener.h: Removed.
* Source/WebKitLegacy/win/WebGeolocationPolicyListener.cpp: Removed.
* Source/WebKitLegacy/win/WebGeolocationPolicyListener.h: Removed.
* Source/WebKitLegacy/win/WebGeolocationPosition.cpp: Removed.
* Source/WebKitLegacy/win/WebGeolocationPosition.h: Removed.
* Source/WebKitLegacy/win/WebHTMLRepresentation.cpp: Removed.
* Source/WebKitLegacy/win/WebHTMLRepresentation.h: Removed.
* Source/WebKitLegacy/win/WebHistory.cpp: Removed.
* Source/WebKitLegacy/win/WebHistory.h: Removed.
* Source/WebKitLegacy/win/WebHistoryItem.cpp: Removed.
* Source/WebKitLegacy/win/WebHistoryItem.h: Removed.
* Source/WebKitLegacy/win/WebInspector.cpp: Removed.
* Source/WebKitLegacy/win/WebInspector.h: Removed.
* Source/WebKitLegacy/win/WebJavaScriptCollector.cpp: Removed.
* Source/WebKitLegacy/win/WebJavaScriptCollector.h: Removed.
* Source/WebKitLegacy/win/WebKit.resources/Info.plist: Removed.
* Source/WebKitLegacy/win/WebKitCOMAPI.cpp: Removed.
* Source/WebKitLegacy/win/WebKitCOMAPI.h: Removed.
* Source/WebKitLegacy/win/WebKitClassFactory.cpp: Removed.
* Source/WebKitLegacy/win/WebKitClassFactory.h: Removed.
* Source/WebKitLegacy/win/WebKitDLL.cpp: Removed.
* Source/WebKitLegacy/win/WebKitDLL.h: Removed.
* Source/WebKitLegacy/win/WebKitGraphics.cpp: Removed.
* Source/WebKitLegacy/win/WebKitGraphics.h: Removed.
* Source/WebKitLegacy/win/WebKitLogging.cpp: Removed.
* Source/WebKitLegacy/win/WebKitLogging.h: Removed.
* Source/WebKitLegacy/win/WebKitMessageLoop.cpp: Removed.
* Source/WebKitLegacy/win/WebKitMessageLoop.h: Removed.
* Source/WebKitLegacy/win/WebKitPrefix.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFImage.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFImage.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFImageInternal.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFView.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFView.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WKCACFViewPrivate.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WebKitQuartzCoreAdditions.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/API/WebKitQuartzCoreAdditionsBase.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CAD3DRenderer.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CAD3DRenderer.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CAView.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CAView.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CVDisplayLink.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CVDisplayLink.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/CVDisplayLinkClient.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/Image.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/Image.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/ImageConversion.cpp: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/ImageConversion.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/InvertColorPS.hlsl: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/WebKitQuartzCoreAdditionsInternal.h: Removed.
* Source/WebKitLegacy/win/WebKitQuartzCoreAdditions/WebKitQuartzCoreAdditionsPrefix.h: Removed.
* Source/WebKitLegacy/win/WebKitStatistics.cpp: Removed.
* Source/WebKitLegacy/win/WebKitStatistics.h: Removed.
* Source/WebKitLegacy/win/WebKitStatisticsPrivate.h: Removed.
* Source/WebKitLegacy/win/WebKitSystemBits.cpp: Removed.
* Source/WebKitLegacy/win/WebKitSystemBits.h: Removed.
* Source/WebKitLegacy/win/WebLocalizableStrings.cpp: Removed.
* Source/WebKitLegacy/win/WebLocalizableStrings.h: Removed.
* Source/WebKitLegacy/win/WebMutableURLRequest.cpp: Removed.
* Source/WebKitLegacy/win/WebMutableURLRequest.h: Removed.
* Source/WebKitLegacy/win/WebNavigationData.cpp: Removed.
* Source/WebKitLegacy/win/WebNavigationData.h: Removed.
* Source/WebKitLegacy/win/WebNodeHighlight.cpp: Removed.
* Source/WebKitLegacy/win/WebNodeHighlight.h: Removed.
* Source/WebKitLegacy/win/WebNotification.cpp: Removed.
* Source/WebKitLegacy/win/WebNotification.h: Removed.
* Source/WebKitLegacy/win/WebNotificationCenter.cpp: Removed.
* Source/WebKitLegacy/win/WebNotificationCenter.h: Removed.
* Source/WebKitLegacy/win/WebPreferenceKeysPrivate.h: Removed.
* Source/WebKitLegacy/win/WebPreferences.cpp: Removed.
* Source/WebKitLegacy/win/WebPreferences.h: Removed.
* Source/WebKitLegacy/win/WebResource.cpp: Removed.
* Source/WebKitLegacy/win/WebResource.h: Removed.
* Source/WebKitLegacy/win/WebScriptObject.cpp: Removed.
* Source/WebKitLegacy/win/WebScriptObject.h: Removed.
* Source/WebKitLegacy/win/WebScriptWorld.cpp: Removed.
* Source/WebKitLegacy/win/WebScriptWorld.h: Removed.
* Source/WebKitLegacy/win/WebSecurityOrigin.cpp: Removed.
* Source/WebKitLegacy/win/WebSecurityOrigin.h: Removed.
* Source/WebKitLegacy/win/WebSerializedJSValue.cpp: Removed.
* Source/WebKitLegacy/win/WebSerializedJSValue.h: Removed.
* Source/WebKitLegacy/win/WebTextRenderer.cpp: Removed.
* Source/WebKitLegacy/win/WebTextRenderer.h: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallenge.cpp: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallenge.h: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallengeSender.cpp: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallengeSender.h: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallengeSenderCFNet.cpp: Removed.
* Source/WebKitLegacy/win/WebURLAuthenticationChallengeSenderCurl.cpp: Removed.
* Source/WebKitLegacy/win/WebURLCredential.cpp: Removed.
* Source/WebKitLegacy/win/WebURLCredential.h: Removed.
* Source/WebKitLegacy/win/WebURLProtectionSpace.cpp: Removed.
* Source/WebKitLegacy/win/WebURLProtectionSpace.h: Removed.
* Source/WebKitLegacy/win/WebURLResponse.cpp: Removed.
* Source/WebKitLegacy/win/WebURLResponse.h: Removed.
* Source/WebKitLegacy/win/WebUserContentURLPattern.cpp: Removed.
* Source/WebKitLegacy/win/WebUserContentURLPattern.h: Removed.
* Source/WebKitLegacy/win/WebView.cpp: Removed.
* Source/WebKitLegacy/win/WebView.h: Removed.
* Source/WebKitLegacy/win/WebWorkersPrivate.cpp: Removed.
* Source/WebKitLegacy/win/WebWorkersPrivate.h: Removed.
* Source/WebKitLegacy/win/storage/WebDatabaseProvider.cpp: Removed.

Canonical link: https://commits.webkit.org/260397@main
